contracts/BnomialNFT.sol

```diff
@@ -24,12 +25,15 @@ contract BnomialNFT is ERC721, Ownable {
         return _tokenIdCounter.current();
     }
 
-    function mint(address to, uint256 badge) external onlyOwner {
+    function mint(address to) external {
```

Comment:
There is one problem with this implementation - everybody will be able to mint badges to any address. I think we should add an additional `require` to check if the owner is minting or if `msg.sender` is the same as `to`.

Reply:
yes, you're right, I'll add the validation